storage/tianmu/system/configuration.cpp

```diff
@@ -57,6 +57,8 @@ unsigned int tianmu_sysvar_delete_or_update_threads;
 my_bool tianmu_sysvar_compensation_start;
 my_bool tianmu_sysvar_filterevaluation_speedup;
 my_bool tianmu_sysvar_groupby_speedup;
+unsigned int tianmu_sysvar_groupby_parallel_degree;
+unsigned long long tianmu_sysvar_groupby_parallel_rows_minimum;
```

Comment:
Maybe percentage is a good choice.

Reply:
This value does have to be debated, but setting the number of rows is the simplest and most straightforward for the user to understand